src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -19,4 +19,6 @@
 -->
 ## Current Release
 
-## Version 2.3.0
\ No newline at end of file
+## Version 2.3.0
+
+* Added storage properties to threat detection cmdlets
```

Comment:
@yoavrubin please move the comment under the "Current Release" section; the "Version 2.3.0" section contains changes made in the previous release, and the "Current Release" section contains changes that will be made for the next Sql release.